migrations/sql/V2019.04.25.11.30__add_variance_applications_metadata.sql

```diff
@@ -0,0 +1,105 @@
+CREATE TABLE variance_application_status_code (
+    variance_application_status_code character varying(3)                   NOT NULL,
+    description                      character varying(100)                 NOT NULL,
+    active_ind                       boolean                  DEFAULT true  NOT NULL,
+    create_user                      character varying(60)                  NOT NULL,
+    create_timestamp                 timestamp with time zone DEFAULT now() NOT NULL,
+    update_user                      character varying(60)                  NOT NULL,
+    update_timestamp                 timestamp with time zone DEFAULT now() NOT NULL,
+
+    UNIQUE(variance_application_status_code)
+);
+
+ALTER TABLE variance_application_status_code OWNER TO mds;
+
+COMMENT ON TABLE variance_application_status_code IS 'The valid options for a variance / variance application.';
+
+INSERT INTO variance_application_status_code (
+    variance_application_status_code,
+    description,
+    create_user,
+    update_user
+)
+VALUES
+    ('REV', 'In Review', 'system-mds', 'system-mds'),
+    ('NAP', 'Not Applicable', 'system-mds', 'system-mds'),
+    ('APP', 'Approved', 'system-mds', 'system-mds'),
+    ('DEN', 'Denied', 'system-mds', 'system-mds');
+
+
+ALTER TABLE variance
+ADD COLUMN variance_application_status_code character varying(3) DEFAULT 'REV' NOT NULL,
```

Comment:
hmmm, do we want variances and variance applications do be in the same table? I know one precedes the other..... I would defer to @klefler or @itchison on their recommendation.

Reply:
The decision to put them in one table was the result of multiple conversations that included Kristin.

Reply:
As long as the record stays intact and we don't create a second record for the same variance that is in a different status, having them in one table is preferable IMO.

Reply:
Sounds like there was good discussion, and there won't be any duplication, thanks Nathan.